phpunit/class-wp-rest-menus-controller-test.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Unit tests covering WP_REST_Menus_Controller functionality
+ *
+ * @package    Gutenberg
+ */
+
+/**
+ * Core class used to manage menus via the REST API.
+ *
+ * @see WP_REST_Controller
+ */
+class WP_REST_Menus_Controller_Test extends WP_REST_Menus_Controller {
+	public function test_it_allows_batch_requests_when_updating_menus() {
```

Comment:
Move this method to [class-rest-nav-menus-controller-test.php](https://github.com/WordPress/gutenberg/blob/trunk/phpunit/class-rest-nav-menus-controller-test.php) and delete this class.

Reply:
I thought the test class must be named according to the class it is testing.
`/__experimental/menus/(?P<id>[\d]+)` endpoint is defined inside the `WP_REST_Menus_Controller`.
Shouldn't we name the test class `WP_REST_Menus_Controller_Test` then?

Reply:
@anton-vlasenko ideally that would be the case. In reality, we don't follow this structure in tests at the moment. The tests for that endpoint are in the file @spacedmonkey mentioned so it makes sense to add new tests there.

As for why we have such a naming structure, I am not sure. Perhaps it's to avoid name conflicts when these classes are later migrated to WP core? @gziolo might have more context here.

Reply:
When me and Tim migrate this to core, I will make sure to name the test class correctly

Reply:
Thank you for the explanation, @adamziel .
I was just curios why we don't follow that structure.
> I will make sure to name the test class correctly